Src/CZICmd/executePlaneScan.cpp

```diff
@@ -37,7 +35,8 @@ class CExecutePlaneScan : public CExecuteBase
             cache_context.prune_options.maxMemoryUsage = max_cache_size;
         }
 
-        IntSize tileSize = { 512, 512 };
+        const IntSize tileSize = { get<0>(options.GetTileSizeForPlaneScan()), get<1>(options.GetTileSizeForPlaneScan()) };
```

Comment:
Could this be changed to cache `options.GetTileSizeForPlaneScan()`? We know options here will just give back the appropriate underlying value but it looks strange to be to invoke this method twice for each item in tuple.